bambi/models.py

```diff
@@ -1070,6 +1085,13 @@ def intercept_term(self):
         else:
             return None
 
+    @property
+    def offset_terms(self):
+        """Return dict of all and only offset effects in model."""
```

Comment:
Wording seems strange. Whats all and only offset effects?

Reply:
Yeah, I agree it's weird. I'm updating it now.